packages/react/src/adapters/ButtonAdapter.js

```diff
@@ -45,7 +45,8 @@ function ButtonAdapter(props) {
           />
           <MapsPropToMethod value={props.disabled} {...adapterProps}>
             {(instance, value) =>
-              value ? instance.disable() : instance.enable()}
+              value ? instance.disable() : instance.enable()
```

Comment:
Any changes to JS files are a result of the `prettier` package updating, and autofixing the lint rules.